modules/ossm-manual-sidecar-injection.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-manual-sidecar-injection_{context}"]
+= Using manual sidecar injection
+
+Manual injection of the sidecar is supported using the upstream Istio community `istioctl` command.
+
+[NOTE]
+====
+When you use manual sidecar injection, ensure you have access to a running cluster so the correct configuration can be obtained from the istio-sidecar-injector configmap within the istio-system namespace.
+====
+
+.Prerequisites
+
+* Download the appropriate link:https://github.com/istio/istio/releases/tag/1.1.8[installation] for your OS
```

Comment:
No GitHub links without clearance from support.

Reply:
@knrc and @brian-avery Please review Kathryn's comment on GitHub links. Do we have clearance for support to link to GitHub here?

Reply:
This link is to an upstream tool we do not support and is only for those wishing to use manual injection during TP.  For GA this option will be removed since we will only be supporting automatic injection and CNI integration.

Reply:
I am leaving this link here with the knowledge that we are still in TP and it will be removed for GA.